src/utils/middleware.ts

```diff
@@ -80,6 +79,11 @@ export const getSpidStrategyOptionsUpdater = (
     )
     .concat([
       fetchIdpsMetadata(serviceProviderConfig.spidCieUrl, CIE_IDP_IDENTIFIERS)
+    ])
+    .concat([
```

Comment:
spidTestentUrl must be optional so I'm expeting here a check as for validator:

`NonEmptyString.is(serviceProviderConfig.spidTesetenvUrl)`

Is it possible to make spidCieUrl optional as well ? I've changed my mind
(this makes this package reusable in context where there's no need to have CIE login)

Reply:
Optional only inside `io-spid-commons` or even on `io-backend`?

Reply:
fixed in 831de37

Reply:
io-spid-commons (isn't the same ? I mean, the IServiceproviderConfig)

Reply:
I mean that into `io-backend` the ENV vars for `CIE_METADATA_URL` is required and  `SPID_TESTENV_URL` if missing use default value.

Reply:
in the backend it's ok ta maintain the current approach